virtme/commands/run.py

```diff
@@ -1190,7 +1190,7 @@ def do_it() -> int:
         with open("/proc/sys/fs/nr_open", encoding="utf-8") as file:
             nr_open = file.readline().strip()
             kernelargs.append(f"nr_open={nr_open}")
-    except FileNotFoundError:
+    except (FileNotFoundError, PermissionError):
         pass
```

Comment:
```suggestion
        pass
    else:
        kernelargs.append(f"nr_open={nr_open}")
```

Unrelated, but this would be a little bit cleaner, but doesn't really matter here :)